src/rpc/rawtransaction.cpp

```diff
@@ -639,16 +639,37 @@ static RPCHelpMan combinerawtransaction()
 {
 
     UniValue txs = request.params[0].get_array();
+
+    // Can't merge < 2 items
+    if (txs.size() < 2) {
+        throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "Missing transactions");
+    }
+
     std::vector<CMutableTransaction> txVariants(txs.size());
 
-    for (unsigned int idx = 0; idx < txs.size(); idx++) {
+    for (unsigned int idx {0}; idx < txs.size(); idx++) {
```

Comment:
```suggestion
    for (unsigned int idx{0}; idx < txs.size(); idx++) {
```
nit: don't need to add the extra space

Reply:
It might be helpful to consider giving a `Concept ACK` or `NACK` before the review. I came across this idea [here](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#code-review).

Reply:
Done

Reply:
Fixed nit